pages/jobs-form.tsx

```diff
@@ -1,12 +1,15 @@
 import Layout from "../components/Layout";
 import FormField from "../components/FormField";
 
+const pageDescription =
+  "Add a job opportunity and have it featured on the Jobs Board to get in touch with local talent.";
```

Comment:
I think we can probably just re-use the page description as the secondary description on most of these pages. So it'd be "Submit a job to be featured on the Code Youngstown Job Board" for both.